meal_plan_overview/meal_plan_overview.php

```diff
@@ -35,46 +47,88 @@
         </div>
     </div>
 </div>
+<div id="overlay" class="overlay">
+    <div class="d-flex justify-content-center">
+        <div class="spinner-grow text-primary" role="status" style="width: 3rem; height: 3rem; z-index: 20;">
+        </div>
+    </div>
+</div>
 <main class="d-flex align-items-center justify-content-center card-design">
-    <div>
+    <div id="anchor" style="display:none">
         <div class="d-flex align-items-center justify-content-center">
-            <h1 class="m-3" style="color:white;">Meal plan overview</h1>
+            <h1 class="m-3">Meal plan overview</h1>
```

Comment:
I would suggest Meal Plan Overview. It looks better and will blend better in with the rest if the website